test.Dockerfile

```diff
@@ -9,6 +9,8 @@ WORKDIR /deso/src
 RUN git clone https://github.com/deso-protocol/core.git
 
 WORKDIR /deso/src/core
+ARG CORE_BRANCH=main
+RUN if ["$CORE_BRANCH" != "main"]; then git checkout main ; else (git branch -D "$CORE_BRANCH" || true) && (git checkout -b "$CORE_BRANCH" origin/"$CORE_BRANCH" || true); fi
```

Comment:
Do we want these changes included?

Reply:
will scrap this for now - deal with it later